native/kotlin/example/composeApp/src/commonMain/kotlin/rs/wordpress/example/shared/ui/login/LoginScreen.kt

```diff
@@ -24,7 +24,7 @@ fun LoginScreen(authenticateSite: (String) -> Unit) {
             verticalArrangement = Arrangement.Center,
             modifier = Modifier.fillMaxSize(),
         ) {
-            var siteUrl by remember { mutableStateOf("boldly-inner.jurassic.ninja") }
+            var siteUrl by remember { mutableStateOf("magnificent-funnel.jurassic.ninja") }
```

Comment:
Might be worth using a `wpmt.co` URL for this so it doesn't have to change?

Reply:
That's a good point. I'll try to address that in my upcoming Kotlin related PR.

Reply:
Addressed [here](https://github.com/Automattic/wordpress-rs/pull/738/files#r2094181097).